dist/scss/wds-components/_avatar.scss

```diff
@@ -1,5 +1,102 @@
+@mixin wds-avatar($diameter: 30px, $color: $wds-color-black, $color-hover: $wds-color-link, $color-background: $wds-color-white) {
+	$border-width: 2px;
+	$inner-border-width: 2px;
+	@if($diameter >= 48px) {
+		$border-width: 3px;
+		$inner-border-width: 3px;
+	}
+	@if($diameter >= 120px) {
+		$border-width: 4px;
+		$inner-border-width: 6px;
+	}
+
+	@include square($diameter);
+
+	a {
+		color: $color;
+	}
+
+	a:hover {
+		color: $color-hover;
+	}
+
+
+	.wds-avatar__image {
+		background-color: $color-background;
+		border: solid $border-width $color;
+
+		&:hover {
+			border-color: $color-hover;
+		}
+	}
+
+	svg.wds-avatar__image {
+		fill: $color;
```

Comment:
these could be set on .wds-avatar__image
browser would just ignore it if it's not svg